external_libraries/oscpack_build.cpp

```diff
@@ -1,8 +1,8 @@
-#include <boost/detail/endian.hpp>
+#include <boost/predef.h>
 
-#ifdef BOOST_LITTLE_ENDIAN
+#ifdef BOOST_ENDIAN_LITTLE_BYTE
 #define OSC_HOST_LITTLE_ENDIAN
-#elif defined(BOOST_BIG_ENDIAN)
+#elif defined(BOOST_ENDIAN_BIG_BYTE)
```

Comment:
[per documentation](https://www.boost.org/doc/libs/1_73_0/libs/predef/doc/index.html#_using_the_predefs), these are always defined, but nonzero, so you should use #if and not #ifdef

Reply:
Done in e48ffe6